Meridian59.DebugUI/DebugForm.cs

```diff
@@ -85,6 +91,17 @@ public DebugForm()
             guildListViewer.PacketSend += new GameMessageEventHandler(gamePacketViewer_PacketSend);
         }
 
+        private void AdminInfoOnWatchObjectAdded(AdminWatchObjectEventHandlerArgs args)
+        {
+            ObjectEditor oe = new ObjectEditor(args.AdminObject);
+            oe.Show();
+        }
+
+        public void HandleAdminMessage(AdminMessage Message)
```

Comment:
This event still exists in the latest version with a new name, it ties the debug form to the AdminData object for showing a new form when needed.